README.md

```diff
@@ -85,16 +85,17 @@ showInstance.getDetails(showKey: "yourShowKey") { result in
 }
 ```
 
-#### `getProducts(showKey:completion:)`
+#### `getProducts(showKey:isEntrance:completion:)`
```

Comment:
Just a suggestion: I think instead of calling it `isEntrance`, `preLive: true/false` or `live` would be better for clarity.
it's no big deal tho, Feel free to ignore!

Reply:
@daman-talkshoplive  Thanks for suggestion. 

@andrea-talkshoplive  would you please allow us to rename the param as I think `preLive` makes more sense?